tests/test_geometries.py

```diff
@@ -111,9 +115,16 @@ def test_line_string_invalid_coordinates(coordinates):
 @pytest.mark.parametrize(
     "coordinates",
     [
+        # One line, two points, no Z
         [[(1.0, 2.0), (3.0, 4.0)]],
+        # One line, two points, has Z
         [[(0.0, 0.0, 0.0), (1.0, 1.0, 1.0)]],
+        # One line, three points, no Z
         [[(1.0, 2.0), (3.0, 4.0), (5.0, 6.0)]],
+        # Two lines, two points each, no Z
```

Comment:
I believe there was a mismatch not being caught without these tests. There was a join with just comma and not comma + space.